backend/webapi.tests/Features/Parties/ProfileStatus.PrimaryCareRostering.cs

```diff
@@ -0,0 +1,58 @@
+namespace PidpTests.Features.Parties;
+
+using FakeItEasy;
+using Xunit;
+
+using static Pidp.Features.Parties.ProfileStatus;
+using static Pidp.Features.Parties.ProfileStatus.Model;
+using Pidp.Infrastructure.HttpClients.Plr;
+using PidpTests.TestingExtensions;
+
+public class ProfileStatusPrimaryCareRosteringTests : ProfileStatusTest
+{
+    [Theory]
+    [MemberData(nameof(PlrSuccessTestData))]
+    public async void HandleAsync_RosteringProviderRoleAuthorized_Locked(PlrStandingsDigest digest)
```

Comment:
I think the names of these test methods haven't changed since you updated the status logic.